src/Sylius/Component/Attribute/AttributeType/SelectAttributeType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * Class SelectAttributeType
+ * @package Sylius\Component\AttributeBundle\AttributeType
+ */
+class SelectAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'select';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return 'jsonArray';
```

Comment:
*Primo*: this value should be added as constant into ``AttributeValueInterface``
*Secundo*: isn't ``json`` enough?